spec/features/admin/organizations/index_spec.rb

```diff
@@ -1,34 +1,44 @@
 require 'rails_helper'
 
 RSpec.feature 'Organization management', type: :feature do
-  let!(:owner) { create(:admin) }
-  let!(:permissions) { create(:user_permission, user: owner, permission_slug: :manage_organizations) }
-  let!(:ihmi) { create(:organization, name: "IHMI") }
-  let!(:path) { create(:organization, name: "PATH") }
+  let!(:owner) {create(:admin)}
+  let!(:permissions) {create(:user_permission, user: owner, permission_slug: :manage_organizations)}
+  let!(:ihmi) {create(:organization, name: "IHMI")}
+  let!(:path) {create(:organization, name: "PATH")}
```

Comment:
Looks like we've edited the spacing here (probably an editor configuration issue, not a big deal). Could we restore the previous spacing?